### PR TITLE
backends: Add set_sshserial_serialdev() and use it

### PR DESCRIPTION
**Pull request by pevik**
DRY

### CODE REVIEW

**Comment by pevik** [comment]
@dzedro @ldevulder @foursixnine @okurz FYI

File: lib/Utils/Backends.pm [hunk 1]
```diff
@@ -55,6 +56,12 @@ our %EXPORT_TAGS = (
     BACKEND  => (BACKEND)
 );
 
+sub set_sshserial_serialdev {
```
Comment (czerw) [inline lib/Utils/Backends.pm]: What about `set_sshserial_dev` ? To make it shorter.

Comment (pevik) [reply lib/Utils/Backends.pm]: Good point, changed. Thanks for a review.

**Comment by ldevulder** [comment]
LGTM

File: lib/Utils/Backends.pm (hunk 1 above)
Comment (foursixnine) [reply lib/Utils/Backends.pm]: I'd go for `set_ssh_serial_dev` Just for consistency but it's not really important

Comment (dzedro) [reply lib/Utils/Backends.pm]: Isn't all this handled by select_serial_terminal ? Why we needanother `set_sshserial_serialdev` and keep `use_ssh_serial_console` ?!

**Comment by pevik** [comment]
OK, now renamed (sorry, forget to push)

**Comment by foursixnine** [comment]
@pevik is this ready?

**Comment by pevik** [comment]
IMHO yes, I can rename function if you wish.

**Comment by foursixnine** [comment]
> IMHO yes, I can rename function if you wish.

Naaa :) That was just a suggestion :P, waiting for @dzedro to merge then...

**Comment by dzedro** [comment]
I don't have any suggestions, I'm just confused about existence of `use_ssh_serial_console` and now creating new `set_sshserial_dev`. I thought we want to get rid of this `use_ssh_serial_console`.
If it's ready to merge, just do it.

**Comment by pevik** [comment]
> I don't have any suggestions, I'm just confused about existence of `use_ssh_serial_console` and now creating new `set_sshserial_dev`. I thought we want to get rid of this `use_ssh_serial_console`.

Yep, that would be ideal. `set_sshserial_dev` is just to remove duplicity, so once somebody decides to dig deep enough to clean the mess to remove `use_ssh_serial_console`, he'll has a bit easier search.

**Pull request merged**